packages/rainbowkit/src/chains/configureChains.ts

```diff
@@ -1,66 +1,36 @@
 import { providers } from 'ethers';
+import { ChainProvider, configureChains as wagmiConfigureChains } from 'wagmi';
 import { Chain } from '../components/RainbowKitProvider/RainbowKitChainContext';
-import { ApiProvider } from './apiProviders/ApiProvider';
+
+type ConfigureChainsConfig = {
+  stallTimeout?: number;
+} & (
+  | {
+      targetQuorum?: number;
+      minQuorum?: never;
+    }
+  | {
+      targetQuorum: number;
+      minQuorum?: number;
+    }
+);
 
 export const configureChains = <
```

Comment:
We chatted about this offline — posting here for visibility.

This `configureChains` function in RainbowKit now wraps the same function from wagmi. The only reason this has been left in RainbowKit is because we have our own `Chain` type that extends wagmi's. We're going to look into whether we can make wagmi's function generic so that we don't need to provide our own wrapper.